cmd/k8s-node-labeller/README.md

```diff
@@ -79,6 +79,10 @@ While container scheduling via label selector works for heterogeneous cluster, i
 
 ## TODOs
 
+## Disclaimer
```

Comment:
Instead of a separate disclaimer section, adding a sentence or two on line 18 is probably better